### ISSUE TITLE
Design principles workshop

### ISSUE THREAD

**Issue by lfrohlich** [raft-tech/TANF-app#252]
As a TDP designer, I want to have clear design principles so that I can evaluate if my work is aligned with our vision and what we've heard from users. 

---

We'd like to finalize our [draft design principles](https://app.mural.co/t/officeoffamilyassistance2744/m/gsa6/1592321236180/a3b53d009801173476763833fd507f0d18e23911), and discuss how they can be used to inform and guide future design work.

AC:
-  Design principles are documented in the wiki

Tasks:
- [x] Review existing design principles
- [x] Design workshop with 18F, OFA and Raft to revisit design principles and discuss how they can inform future work
- [x] Have workshop
- [x] Document design principles in wiki (update on [Design Guide page](https://github.com/HHS/TANF-app/wiki/Design-Guide) or maybe add to Product Strategy section?)
- [x] Socialize design principles with broader team.
- [x] Incorporate design principles into design ACs

DoD:
- [x] Documentation: Design principles are documented and shared with PO and team
- [x] @lfrohlich has reviewed and signed off

**Comment by cmajel** [comment]
A few updates here:

* @reitermb and I designed a workshop format. [Planning doc here](https://teams.microsoft.com/l/file/249AA6AA-FD78-4730-9A9D-05053AD3655C?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FTeam%20Meetings%20%2B%20Presentations%2F2020%20Sept%20Design%20Principles%20workshop%2FDesign%20principles%20planning.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287), and [overview of design principles](https://teams.microsoft.com/l/file/704DF7B7-6955-4265-B578-BADD22413927?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=pptx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FTeam%20Meetings%20%2B%20Presentations%2F2020%20Sept%20Design%20Principles%20workshop%2F2020.09.25%20Design%20Principles%20Workshop.pptx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287) here. 
* We had a workshop with OFA, Raft, OFA regional offices and Tribal TANF folks on Fri 9/25. [Mural here](https://app.mural.co/t/gsa6/m/gsa6/1600965012810/458a31d9f962e6542e68bf87ba93186cdb7003b9). We got some great context from the group.
* I've started grouping what we heard into themes.
* Next steps
  * Share proposed design principles with the team, work through ways they could be enforced or weakened by working patterns
  * Transfer mural ownership to OFA 
  * Remaining tasks

**Comment by cmajel** [comment]
Sent draft design principles to workshop participants for feedback.

**Comment by cmajel** [comment]
@lfrohlich I'm passing this to you for review. I've updated the wiki with a new Design principles page in the Design section, which includes:
* A write up on what design principles are
* How we got to these principles
* The initial principles as defined in the mural from our workshop

I also included some starter prompts for how the team could embody these principles in their work. Do these feel right? I can schedule some time with the team to flesh these out further this week (or we could pass it to another team member)

I also opened #326 which adds reviewing work alongside design principles as a part of our AC process.

ccing @SelenaJV and @shubhi-raft for thoughts as well.
